<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Blowfish/XOR Encryption Example</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/dialog-password.png</normaloff>:/images/dialog-password.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Key:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="keyEdit">
        <property name="text">
         <string>ABCD</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="setKeyBtn">
        <property name="text">
         <string>Set Key</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Input:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="inputEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Output:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="outputEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QComboBox" name="selEncryption">
        <item>
         <property name="text">
          <string>Blowfish</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>XOR</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="selDataType">
        <item>
         <property name="text">
          <string>Rich Text (html)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Simple Text</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="encryptBtn">
        <property name="text">
         <string>Encrypt</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/object-locked.png</normaloff>:/images/object-locked.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="decryptBtn">
        <property name="text">
         <string>Decrpyt</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/object-unlocked.png</normaloff>:/images/object-unlocked.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>428</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
